symbols/overlay29.yml

```diff
@@ -1758,7 +1847,7 @@ overlay29:
         
         r0: attacker pointer
         r1: defender pointer
-    - name: ApplyGummiBoosts
+    - name: ApplyGummiBoostsDungeon
```

Comment:
Even though it's a long name, let's call this ApplyGummiBoostsDungeonMode to be consistent with ApplyGummiBoostsGroundMode

Reply:
Makes sense